.github/workflows/fedora-copr-build.yml

```diff
@@ -190,22 +190,22 @@ jobs:
           done
 
       - name: "Delete target Copr project at ${{ env.project_target }} before forking to it"
-        if: ${{ env.yesterdays_project_exists == 'yes' && env.target_project_exists == 'yes' }}
+        if: ${{ matrix.forked_repo && env.yesterdays_project_exists == 'yes' && env.target_project_exists == 'yes' }}
         run: |
           copr-cli delete "${{ env.project_target }}"
           # Give Copr some time to process the deletion, to avoid race conditions with forking.
           # TODO: Keep and eye on https://github.com/fedora-copr/copr/issues/2698 if there's a better way to handle this.
           sleep 1m
 
       - name: "Fork Copr project from ${{ env.project_yesterday }} to ${{ env.project_target }}"
-        if: ${{ env.yesterdays_project_exists == 'yes' }}
+        if: ${{ matrix.forked_repo && env.yesterdays_project_exists == 'yes' }}
         run: |
           copr-cli fork --confirm ${{ env.project_yesterday }} ${{ env.project_target }}
           copr-cli modify --delete-after-days -1 --unlisted-on-hp off ${{ env.project_target }}
 
       - name: "Regenerate repos for target project ${{ env.project_target }}"
         # If yesterday's project didn't exist, we haven't forked and so we don't
         # need to regenerate the repos.
-        if: ${{ env.yesterdays_project_exists == 'yes' }}
+        if: ${{ matrix.forked_repo && env.yesterdays_project_exists == 'yes' }}
```

Comment:
We probably need to regenerate the repo even it if is not forked. Otherwise the content is not updated, no?